pkg/controllers/provisioning/provisioner.go

```diff
@@ -315,23 +317,21 @@ func (p *Provisioner) Schedule(ctx context.Context) (scheduler.Results, error) {
 	s, err := p.NewScheduler(ctx, pods, nodes.Active())
 	if err != nil {
 		if errors.Is(err, ErrNodePoolsNotFound) {
-			logging.FromContext(ctx).Info(ErrNodePoolsNotFound)
+			log.FromContext(ctx).Info(ErrNodePoolsNotFound.Error())
```

Comment:
Should this really be an error? 
```suggestion
			log.FromContext(ctx).Error("nodepool not found", ErrNodePoolsNotFound)
```

Reply:
fixed